Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/ReadBlob.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Text;
+using Azure.Storage.Blobs;
+using Azure;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        /// Read a single file to Azure blob storage using connection string or SAS Token.
+        /// [Documentation](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/tree/main/Frends.AzureBlobStorage.ReadBlob)
+        /// </summary>
+
+        public static Result ReadBlob([PropertyTab] Source source, CancellationToken cancellationToken)
+        {
+            var data = ReadBlobContentAsyncTest(source, cancellationToken);
+            return new Result(data.ToString());
+        }
+
+        private static string ReadBlobContentAsyncTest(Source source, CancellationToken cancellationToken)
+        {
+            BlobClient blob;
+            var uri = $"{source.Uri}/{source.ContainerName}/{source.BlobName}?";
+
+            if (String.IsNullOrEmpty(source.ConnectionString))
```

Comment:
You should implement the selector, where user can select which authentication method he wants to use. Then you can check which method user has selected and use the correct method for that. In both connection string and SAS token cases, you also need to check if user has provided the necessary string/token and throw an error if user hasn't provided the string/token.